src/styles/main.css

```diff
@@ -1,21 +1,255 @@
+.description-tabs {
```

Comment:
there are a few issues with the styles. there is no need to fix them right now but I recommend to check out this playlist https://youtube.com/playlist?list=PL4-IK0AVhVjNDRHoXGort7sDWcna8cGPA